lib/core/nuxt.js

```diff
@@ -120,7 +120,28 @@ export default class Nuxt {
 
         const _host = host === '0.0.0.0' ? 'localhost' : host
         // eslint-disable-next-line no-console
-        console.log('\n' + chalk.bgGreen.black(' OPEN ') + chalk.green(` http://${_host}:${port}\n`))
+        console.log('\n' + chalk.bgGreen.black(' OPEN ') + chalk.green(` http://${_host}:${port}`))
+
+        if (host === '0.0.0.0') {
+          const ifaces = require('os').networkInterfaces()
+
+          Object.keys(ifaces).forEach((ifName) => {
+            let ifAliasNum = 0
+            let ifAlias = ''
+            ifaces[ifName].forEach((iface, index) => {
+              if (iface.family === 'IPv4' && iface.internal === false) {
+                if (ifAliasNum > 0) {
+                  ifAlias = `:${ifAliasNum}`
+                }
+                // eslint-disable-next-line no-console
+                console.log(chalk.bgGreen.black(` OPEN (${ifName}${ifAlias}) `) + chalk.green(` http://${iface.address}:${port}`))
```

Comment:
While this is a really good information, but maybe we can use greyed lines indicating nuxt is listening on this interface and IPs too. Leaving that OPEN message to be a single one for easy click and open and reducing users confusing with many ips.

Reply:
Would you be ok that the OPEN statement for an unspecified address would show as 0.0.0.0 or do you still feel that it should list as localhost?
Imho the OPEN statement should reflect the real user configuration, as the localhost change for 0.0.0.0 is really a specific use-case when you are developing on the same machine as you run the server. As for me thats not the case I will never have any use for that localhost link

Reply:
One common use case for using 0.0.0.0 in local envs is for testing on mobile within local network. So this PR still makes sense. The only problem is that `OPEN http://0.0.0.0:3000` is confusing because it won't work!